discovery_utils/getters/overton.py

```diff
@@ -0,0 +1,1536 @@
+
+        # Convert boolean fields
+        if "authors_are_organization" in df.columns:
+            df["authors_are_organization"] = df["authors_are_organization"].astype(bool)
+
+        self.logger.debug(f"Processed {len(df)} documents successfully with {len(df.columns)} fields")
+        return df
+
+    def _truncate_content(self, content: str, max_length: int = 1000) -> str:
+        """Truncate content to specified length with ellipsis.
+
+        Args:
+            content (str): Content to truncate
+            max_length (int): Maximum length. Defaults to 1000.
+
+        Returns:
+            str: Truncated content
+        """
+        if not content:
+            return "No content available"
+
+        if len(content) <= max_length:
+            return content
+
+        # Truncate and add ellipsis
+        return content[:max_length].rstrip() + "..."
+
+    def _safe_int_conversion(self, value: Any) -> int:
+        """Safely convert value to integer.
+
+        Args:
+            value (Any): Value to convert
+
+        Returns:
+            int: Converted integer or 0 if conversion fails
+        """
+        try:
+            if value is None:
+                return 0
+            return int(float(value))
+        except (ValueError, TypeError):
+            return 0
+
+    def _safe_float_conversion(self, value: Any) -> float:
+        """Safely convert value to float.
+
+        Args:
+            value (Any): Value to convert
+
+        Returns:
+            float: Converted float or 0.0 if conversion fails
+        """
+        try:
+            if value is None:
+                return 0.0
+            return float(value)
+        except (ValueError, TypeError):
+            return 0.0
+
+    def get_facets(self, query: Optional[str] = None, clear_cache: bool = False) -> Dict[str, List[Dict]]:
+        """Get facet information for filtering searches.
+
+        Retrieves available facets and their counts, which can be used
+        to understand the data distribution and build effective filters.
+
+        Args:
+            query (str, optional): Apply query to get facets for specific search.
+                If None, returns general facets.
+            clear_cache (bool, optional): Force refresh of cached facets.
+                Defaults to False.
+
+        Returns:
+            Dict[str, List[Dict]]: Facet information by field name.
+                Each facet contains 'key' and 'doc_count' fields.
+
+        Example:
+            >>> overton = OvertonGetter()
+            >>> facets = overton.get_facets("climate change")
+            >>> countries = facets.get("policy_source_country", [])
+            >>> print(f"Top country: {countries[0]['key']} ({countries[0]['doc_count']} docs)")
+        """
+        # Check if we should use cached facets
+        if self._facets_cache and not clear_cache and not query:
+            return self._facets_cache
+
+        params = {}
```

Comment:
```suggestion
        params = {"show_search_facets": True}
```

Fyi, to get facets, I had to add this parameter